scripts/variantstore/wdl/HailFromWdl.wdl

```diff
@@ -0,0 +1,250 @@
+                                      boot_disk_gb: 10
+                                  }
+    RuntimeAttr runtime_override = select_first([runtime_attr_override, runtime_default])
+
+    String default_script_filename = "filter_VDS_and_shard_by_contig.py"
+
+    command <<<
+        # Prepend date, time and pwd to xtrace log entries.
+        PS4='\D{+%F %T} \w $ '
+        set -o errexit -o nounset -o pipefail -o xtrace
+
+        account_name=$(gcloud config list account --format "value(core.account)")
+
+        pip3 install --upgrade pip
+        pip3 install hail~{'==' + hail_version}
+        pip3 install --upgrade google-cloud-dataproc
+
+        if [[ -z "~{git_branch_or_tag}" && -z "~{submission_script}" ]] || [[ ! -z "~{git_branch_or_tag}" && ! -z "~{submission_script}" ]]
+        then
+            echo "Must specify git_branch_or_tag XOR submission_script"
+            exit 1
+        elif [[ ! -z "~{git_branch_or_tag}" ]]
+        then
+            script_url="https://raw.githubusercontent.com/broadinstitute/gatk/~{git_branch_or_tag}/scripts/variantstore/wdl/extract/~{default_script_filename}"
+            curl --silent --location --remote-name "${script_url}"
+        fi
+
+        if [[ ! -z "~{submission_script}" ]]
+        then
+            script_path="~{submission_script}"
+        else
+            script_path="~{default_script_filename}"
+        fi
+
+        # Generate a UUIDish random hex string of <8 hex chars (4 bytes)>-<4 hex chars (2 bytes)>
+        hex="$(head -c4 < /dev/urandom | xxd -p)-$(head -c2 < /dev/urandom | xxd -p)"
+
+        cluster_name="~{prefix}-~{contig}-hail-${hex}"
+        echo ${cluster_name} > cluster_name.txt
+
+        python3 /app/run_in_hail_cluster.py \
+            --script-path ${script_path} \
+            --account ${account_name} \
+            --num-workers ~{num_workers} \
+            ~{'--worker-machine-type' + worker_machine_type} \
+            --region ~{region} \
+            --gcs-project ~{gcs_project} \
+            --cluster-name ${cluster_name} \
+            --prefix ~{prefix} \
+            --contig ~{contig} \
+            --vds-url ~{vds_url} \
+            --vcf-header-url ~{vcf_header_url} \
+            --bed-url ~{bed_url}
+
+        echo "Complete"
+    >>>
+
+    output {
+        String cluster_name = read_string("cluster_name.txt")
+        File vcf = "~{prefix}.~{contig}.vcf.bgz"
+    }
+
+    runtime {
+        memory: select_first([runtime_override.mem_gb, runtime_default.mem_gb]) + " GB"
+        disks: "local-disk " + select_first([runtime_override.disk_gb, runtime_default.disk_gb]) + " SSD"
+        cpu: select_first([runtime_override.cpu_cores, runtime_default.cpu_cores])
+        preemptible: select_first([runtime_override.preemptible_tries, runtime_default.preemptible_tries])
+        maxRetries: select_first([runtime_override.max_retries, runtime_default.max_retries])
+        # `slim` here to be able to use Java
```

Comment:
Could you clarify this comment?

Reply:
oh whoops this is no longer true, will delete